PR for pre-publication review checklist for all content types

rinkug opened the pull request:
This is a DRAFT PR.

It will resolve Issue [#391](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/issues/391), when completed.

markcmiller86 commented:
The common review criteria is a great list. 

Some other thoughts...

- I don't think we need an *issue* and a *PR*. The whole discussion can occurr here in the PR. So, the issue isn't necessary. Lets close the issue.
- If you want the people to review this that you listed in the issue, go ahead and assign them as reviewers. Don't be shy ;) People can still decline/ignore your request.
- I see a lot of content (sub)types mentioned I was not aware of (listed below).
  - publications
  - tutorials
  - web articles
  - Websites or Organization
  - Book
  - Blogs and original experiences
- In the interest of keeping things simple (and I have a vague recollection we agreed to this already), I thought we'd have just a few content types (Event, Curated Content, Original Article).
- Many of the criteria can have automated checks backing them up
  - in some cases, an automated check will serve only to indicate there is noting agregious about the content. It still requires human approval. But, automated/CI checks would help a lot here. I guess I should bite the bullet and write a test for spell checking.
- Is *licensing* a criteria? Shouldn't *we* be defining what the licensing terms of content posted here is and part of the criteria is that the author indicates their acceptance of those terms, perhaps by checking a box like so
  - [ ] By checking this box, author acknowledges and agrees to licensing terms of content they develop here.
- How can authors be cognizant of and draft content within meeting guidelines of *hidden* markdown formating requirements imposed by the back-end site generation?

rinkug commented:
> The common review criteria is a great list.
> 
> Some other thoughts...
> 
> * I don't think we need an _issue_ and a _PR_. The whole discussion can occurr here in the PR. So, the issue isn't necessary. Lets close the issue.

Issue closed

> * If you want the people to review this that you listed in the issue, go ahead and assign them as reviewers. Don't be shy ;) People can still decline/ignore your request.

Will do

> * I see a lot of content (sub)types mentioned I was not aware of (listed below).
>   
>   * publications
>   * tutorials
>   * web articles
>   * Websites or Organization
>   * Book
>   * Blogs and original experiences
> * In the interest of keeping things simple (and I have a vague recollection we agreed to this already), I thought we'd have just a few content types (Event, Curated Content, Original Article).

We still have only 3 content types for the purpose of workflow (and perhaps other reasons too for example something like directory organization in github?).  But, when it comes to actual content, we do have these different content types as listed above that we need to support. The review checklist will have different criteria for them (for ex: publications will need a DOI, tutorial will need slides/audio/video links, books will need ISBN etc.)

> * Many of the criteria can have automated checks backing them up
>   * in some cases, an automated check will serve only to indicate there is noting agregious about the content. It still requires human approval. But, automated/CI checks would help a lot here. I guess I should bite the bullet and write a test for spell checking.

Yes, definitely, auto checks will help. I am still learning about these so if you know about such tools please add them at the bottom of the document and we can organize it later

> * Is _licensing_ a criteria? Shouldn't _we_ be defining what the licensing terms of content posted here is and part of the criteria is that the author indicates their acceptance of those terms, perhaps by checking a box like so
>   * [ ]  By checking this box, author acknowledges and agrees to licensing terms of content they develop here.

Thats a good point. My original thought in adding licensing to the checklist was to bring it up for discussion for this very purpose and see if we need to consider it.

> * How can authors be cognizant of and draft content within meeting guidelines of _hidden_ markdown formating requirements imposed by the back-end site generation?

I understand and I think authors should meet github markdown requirements to the best they can when they submit the article or PR. The editorial team will deal with the custom hidden markdown changes (which are not too many).

markcmiller86 commented:
I moved everyone to `Reviewer` instead of `Assigned`. The `Assigned` person is the one actually drafting the content. The `Reviewe` persons are the ones who review the work the `Assigned` person(s) do.

pagrubel commented:
Why have the following in the actually in the document?
"This document will be posted in Issue #391 or PR #392, so the document will eventually contain any discussion held at these links.

Prior discussion that touch this topic are outlined in PR #372, Issue #371 and in our new proposed curated content workflow

Before we can develop review checklists for various types of content, we need to have a clear idea of the types of content we expect to support. Our original understanding of our content types was described in the What to Contribute document. Lately, there is another PR/issue (PR #373, Issue #361) in the works which attempts to revise the content types and their attributes. "

The statements are in the top of the discussion in this PR and frankly it confused me between that an issue and a PR. Thanks for closing the issue. 
For the point "Check tone of the article" I assuming that is the document we are reviewing not something a curated content is pointing to. That should have been done in the previous review process. I guess I'm saying this maybe should be Check "Tone" of the document? What are we checking here?

Does body text mention why this paper is interesting, for what audience is may be suitable etc? (this should already have been done by reviewers as well - do we need this here??) I would think this should be done before this step.

rinkug commented:
Document has been updated with more details. I have removed references to "style guide for BSSw content" and moved it to a new document (ContentStyleGuide.md). A separate PR will be initiated for the style guide.
